data/sumitishere.json

```
@@ -21,28 +21,28 @@
     },
     {
       "icon": "FaLinkedin",
```

[Proofreading] `icon` strings are not supposed to end in a comma.
```suggestion
      "icon": "FaLinkedin"
```

---

@CBID2 this is incorrect. The property name does not affect if it should end in a comma so I will mark this as resolved.